cmd/atlas/atlas.go

```diff
@@ -124,7 +124,7 @@ func mongoCLIConfigFilePath() (configPath string, err error) {
 		return configPath, nil
 	}
 
-	if configDir, err := config.OldMongoCLIConfigHome(); err == nil {
```

Comment:
Only flagging recently for my linter as I updated to `v1.45.2`. We want to deprecate for some time before fully removing the code, so disabling the lint check for this line only.